### ISSUE TITLE
Gesture: Ruler helptext is read out multiple times

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#203]
In 2.2.0-dev.18, the help text delivered when the Measure Distance Ruler gains gesture focus is read out three times instead of just once.

**Comment by terracoda** [comment]
@zepumph, I am sorry I do not know why. Could you check that the help text content appears in the code in a single place.

**Comment by terracoda** [comment]
This is a serious problem. It the help text is being read out 6 times now.
Are we using aria-describedby? If so, maybe it doesn't work well on mobile safari? Maybe we should try without it and see what VO does.

**Comment by terracoda** [comment]
@zepumph I marked high priority as I can't use the sim with that kind of repetition going on. And I  do not know how to debug it. (Sorry)

**Comment by zepumph** [comment]
@jessegreenberg  I think this is because we added the down/up listeners, but we are also handling the gesture case on the PDOM side. Likely we want to just have this handled by the general presslistener that we added recently. I'll implement

**Comment by zepumph** [comment]
I am able to reproduce this on 2.2.0-dev.18, but not on master. @terracoda please confirm this is fixed from other work @jessegreenberg and I have been doing in the dev version I will send you this evening.

**Comment by terracoda** [comment]
@zepumph, recent commits tot he ruler interaction have made it much much better over all. I am no longer hearing repeated help text. AND, I get no auto help text (no aria-describedby) after 2 grabs which I find really nice (thanks!).

There is still some quirkyness:
- Distance string in initial grab alert gets cut off.
- Ruler is a bit jumpy, meaning it can just move to another pot on the screen when I try and grab it.

 I will open new issues for these as something to watch for.

**Comment by zepumph** [comment]
>Ruler is a bit jumpy, meaning it can just move to another pot on the screen when I try and grab it.


Note that this is a bug from https://github.com/phetsims/gravity-force-lab/issues/191, and should be fixed before publication.
